com.unity.ml-agents/CHANGELOG.md

```diff
@@ -33,6 +33,13 @@ different sizes using the same model. For a summary of the interface changes, pl
 - `GridSensor` has been refactored and moved to main package, with changes to both sensor interfaces and behaviors.
 Exsisting GridSensor created by extension package will not work in newer version. Previously trained models will
 need to be retrained. Please see the Migration Guide for more details. (#5256)
+- Models trained with 1.x versions of ML-Agents will no longer work at inference if they were trained using recurrent neural networks (#5254)
+
+#### ml-agents / ml-agents-envs / gym-unity (Python)
```

Comment:
Is this intentional? It was in 1.9.1, right?

Reply:
No it is not. I think I made a mistake when resolving conflicts in the last commit.